rmc/resources/exac.py

```diff
@@ -33,7 +33,7 @@ def _import_coverage(**kwargs) -> hl.Table:
     return coverage
 
 
-# Files for direct comparison with Kaitlin's code
+## Files for direct comparison with Kaitlin's code
 EXAC_PREFIX = f"{RESOURCE_PREFIX}/GRCh37/exac"
```

Comment:
maybe we should update this file to only contains comparison files (and note that in a docstring at the top of the script), since we only use ExAC data for comparison purposes

Reply:
No I left it as `RESOURCE_PREFIX` because it will always be in GRCh37 and does not change with `CURRENT_BUILD`

Reply:
ah sorry, I meant we should remove the current resources in this script (ExAC coverage/sites) and replace with files we use to compare ExAC o/e and missense badness/MPC scores to their updated gnomAD versions

Reply:
^ we can add these files later, but probably good to delete the current resources now